### DISCUSSION TITLE
Signature

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7265] [Help]
Hello everyone.
Is there anything new concerning the signature component ?

**Comment by andz-bb** [comment]
Hey @rafikApp - the signature component is logged as an open issue here: #4380

You can subscribe to that issue or check back soon to see if any progress has been made. Custom components are also in the roadmap for Q3. This new functionality should allow a signature component to be implemented if it's not yet part of core functionality by then.
